src/util/displayInfo.js

```diff
@@ -5,20 +5,28 @@ L.Control.Info = L.Control.extend({
 
   initialize: function(options) {
     L.Util.setOptions(this, options);
-    this._infoDisplay = L.DomUtil.create('div');
-    this._infoDisplay.classList.add('leaflet-control-info', 'leaflet-bar', 'leaflet-control');
-    this._infoDisplay.style.backgroundColor = '#fff';
-    if(this.options.classname) {
-      this._infoDisplay.classList.add(this.options.classname);
+    if (this.options.classname) {
+      var infoModal = "<div  class='modal fade leaflet-bar leaflet-control-info"+ this.options.classname +"' role='dialog' tapindex='3' aria-hidden='true' >";
     }
-    this._textElement = L.DomUtil.create('span');
-    this._textElement.innerHTML = this.options.text;
-    this._textElement.classList.add('leaflet-control-info-text');
-    this._infoDisplay.appendChild(this._textElement);
-    this._closeButton = L.DomUtil.create('button');
-    this._closeButton.classList.add('leaflet-control-info-button');
-    this._closeButton.innerText = 'close';
-    this._infoDisplay.appendChild(this._closeButton);
+    else {
+      var infoModal = "<div  class='modal fade leaflet-bar leaflet-control-info' role='dialog' tapindex='3' aria-hidden='true' >";
+    }
+    infoModal += "<div class='modal-dialog'>";
```

Comment:
This is really nice and clean. Can you share a screenshot?

Reply:
Yeah Sure
![displayinfocode](https://user-images.githubusercontent.com/55199130/79297686-85362d00-7efc-11ea-8e4c-807a427eea53.png)